watchdog/src/config.rs

```diff
@@ -67,20 +57,12 @@ pub struct Config {
 }
 
 impl Config {
-    /// Creates a new configuration depending on the Bitcoin network.
-    pub fn new() -> Self {
-        match BITCOIN_NETWORK {
-            BitcoinNetwork::Mainnet => Self::mainnet(),
-            BitcoinNetwork::Testnet => Self::testnet(),
-        }
-    }
-
     /// Creates a new configuration for the mainnet.
     pub fn mainnet() -> Self {
         Self {
             bitcoin_network: BitcoinNetwork::Mainnet,
-            blocks_behind_threshold: BLOCKS_BEHIND_THRESHOLD,
-            blocks_ahead_threshold: BLOCKS_AHEAD_THRESHOLD,
+            blocks_behind_threshold: 2,
+            blocks_ahead_threshold: 2,
             min_explorers: MIN_EXPLORERS,
             bitcoin_canister_principal: Principal::from_text(MAINNET_BITCOIN_CANISTER_PRINCIPAL)
                 .unwrap(),
```

Comment:
FYI: 
in the last upgrade config did not have `ApiBitapsComMainnet` endpoint because it was misbehaving and was removed (probably before that).
Just [checked](https://api.bitaps.com/btc/v1/blockchain/block/last) it now and it seems to be working again, so we can add it again to the default config.

https://dashboard.internetcomputer.org/proposal/125325

Reply:
We have that already in this config, so no action required, right?

Reply:
Yes, no action required for re-enabling that explorer that looks ok at the moment.